src/main/java/org/springframework/data/tarantool/core/mapping/BasicTarantoolPersistentProperty.java

```diff
@@ -41,10 +40,18 @@ public BasicTarantoolPersistentProperty(Property property,
 
     @Override
     public String getFieldName() {
-        Field annotationField = AnnotatedElementUtils.findMergedAnnotation(getField(), Field.class);
-
-        if (annotationField != null && StringUtils.hasText(annotationField.value())) {
-            return annotationField.value();
+        java.lang.reflect.Field field = getField();
```

Comment:
Change to class name

Reply:
This is impossible, because we already have the `Field` annotation in this package, we cannot use the `reflect.Field` class, because name clashing will happen